testonly/integration/clientserver.go

```diff
@@ -55,54 +56,54 @@ func listen() (string, net.Listener, error) {
 }
 
 // resetDB drops and recreates the test database.
+// returns a connection to the test database.
```

Comment:
(nit) returns a database URI, not a connection; also need a capital letter at the start of a sentence.